packages/instrumentation/package.json

```diff
@@ -22,9 +22,9 @@
     "typescript": "5.3.3"
   },
   "dependencies": {
-    "@opentelemetry/api": "1.8.0",
-    "@opentelemetry/instrumentation": "0.49.1",
-    "@opentelemetry/sdk-trace-base": "1.22.0"
+    "@opentelemetry/api": "^1.8",
+    "@opentelemetry/instrumentation": "^0.49",
```

Comment:
```suggestion
    "@opentelemetry/instrumentation": "^0.49 || ^0.50",
```

Reply:
thanks!

I appreciate your feedback. I've updated version ranges to include current and previous versions:



<img width="472" alt="image" src="https://github.com/prisma/prisma/assets/7002833/7a8a1c1d-0f97-4eb1-a679-210edb9f114c">


Thanks!